Firebase/Storage/Private/FIRStorageMetadata_Private.h

```diff
@@ -34,6 +34,23 @@ NS_ASSUME_NONNULL_BEGIN
 @property(readwrite) FIRStorageMetadataType type;
 
 /**
+ * The original metadata representation received from the server.
+ */
+@property(copy, nonatomic) NSDictionary *serverState;
```

Comment:
I don't get how this has the server state before we update the metadata, since we don't seem to download that metadata before we update? Or does this only work for updates following fetches?